cEP-0020.md

```diff
@@ -0,0 +1,543 @@
+it a `Thumbs Up` reaction. But in case if the issue is invalid
+or labelled incorrectly the developer will request a maintainer
+on Gitter channel to label it correctly and only then approve
+the issue by giving it a `Thumbs Up`.
+
+An issue should have at least two `Thumbs Up` to be able to
+tracked in this system.
+
+Similarly, if a pull request opened by a newcomer is closing
+or fixing an unverified issue it will not be tracked in this
+system.
+
+### Levels
+
+To increase a user's competitive instinct, we can implement a
+series of levels that confer rank as newcomers become more
+active. A good and straightforward way to award levels in a
+new gamified design is to base them on point thresholds. As
+users earn points, they move up an incremental series of levels.
+To infuse the level with meaning i.e.: a ranking within the
+system, the levels should be named in a way that indicates status.
+
+But the levels wouldn't be enough motivation to go until newcomers
+get some special privileges after completing each level:
+
+Note that the following levels are an example of possible levels
+with suggested permissions based on the example points allocations
+in the previous section.
+
+- Level-I:
+  - Name: Fresher
+  - Points_required = 0
+  - Privileges:
+    - No privileges for freshers
+- Level-II:
+  - Name: Newbie
+  - Points_required = 10
+  - Privileges:
+    - Get assigned to newcomers issue
+- Level-III:
+  - Name: Beginner
+  - Points_required = 20
+  - Privileges:
+    - Get assigned to a difficulty low issue
+- Level-IV:
+  - Name: Intermediate
+  - Points_required = 50
+  - Privileges:
+    - Mark your own PR as wip
+- Level-V:
+  - Name: Cool
+  - Points_required = 100
+  - Privileges:
+    - Mark a PR by another newcomer as wip
+    - Get assigned to difficulty medium issue
+- Level-VII:
+  - Name: Expert
+  - Points_required = 200
+  - Privileges
+    - Assign newcomers to difficulty newcomer issue
+    - Assign newcomers to difficulty low issue
+- Level-VIII:
+  - Name: Legend
+  - Points_required = 350
+  - Privileges
+    - Invite newcomers to org
+    - Assign newcomers to difficulty medium issue
+    - Permission to `ack` and `unack` on the pull requests
+
+Implementation of privileges is beyond scope of the
+gamification system. The gamification system will merely
+expose a list of privileges that each person has, and other
+systems are expected to adapt to use these privileges.
+E.g.: [corobo](https://github.com/coala/corobo)
+
+While implementing privileges, a first manual check should
+be done for testing purpose, and then automated only if there
+is no serious risk or there is a good risk reaction mechanism
+in place.
+
+### Badges
+
+Another way of explicitly nudging a newcomer to action is to
+award badges for completing tasks. Badges enable newcomers to
+follow their performance and compare to other newcomers.
+Badges will be awarded after performing certain activities by
+the newcomers:
+
+- Badge-I:
```

Comment:
calling them Badge-I here suggests that order is important, but badges are not about order.

IMO add the badge Name here, and then omit it from the sublist.